File: 1. Model/Endereco.py


class Endereco:

    def __init__(self, bairro, rua, numRes, telefone):
        self.bairro   = bairro
        self.rua      = rua
        self.numRes   = numRes
        self.telefone = telefone
        





    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getBairro(self):
        return self.bairro

    def getRua(self):
        return self.rua

    def getNumRes(self):
        return self.numRes

    def getTelefone(self):
        return self.telefone


    ##SETS
    def setBairro(self, b):
        self.bairro = b

    def setRua(self, r):
        self.rua = r

    def setNumRes(self, nr):
        self.numRes = nr

    def setTelefone(self, t):
        self.telefone = t
    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Endereco.java
package model;



public class Endereco {

    private String bairro;
    private String rua;

    private int numRes;
    private int telefone;



    public Endereco(String b, String r, int n, int t) {
        this.bairro   = b;
        this.rua      = r;
	    this.numRes   = n;
        this.telefone = t;
    }



    // Gets
    public String getBairro() {
        return this.bairro;
    }

    public String getRua() {
        return this.rua;
    }

    public int getNumRes() {
        return this.numRes;
    }

    public int getTelefone() {
        return this.telefone;
    }



    // Sets
    public void setBairro(String b) {
        this.bairro = b;
    }

    public void setRua(String r) {
        this.rua = r;
    }

    public void setNumRes(int n) {
        this.numRes = n;
    }

    public void setTelefone(int t) {
        this.telefone = t;
    }
}
